package com.SP3x3.managers;

public class PluginManager {
   private static PluginManager instance;

   public static PluginManager getInstance() {
      if (instance == null) {
         instance = new PluginManager();
      }

      return instance;
   }

   public void initialize() {
   }
}